server/@types/dto/index.d.ts

```diff
@@ -84,6 +84,9 @@ declare module 'dto' {
     completedByOtherJobRole?: string
     reviewDate: Date
     notes?: string
+    wasLastReviewBeforeRelease?: boolean
```

Comment:
Needed to add these to the DTO, to carry the properties from the response data of calling the API (IE. The Review Complete page needs to know whether this was the last review and the next review dates)